src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -248,7 +269,7 @@ internal string CreateSearchQuery(string[] fields, string SearchTerm)
         internal void DisposeWriter()
         {
             //We need to check if we are not running Dynamo tests because otherwise parallel test start to fail when trying to write in the same Lucene directory location
-            if (!DynamoModel.IsTestMode)
+            if (!DynamoModel.IsTestMode || currentStorageType == LuceneStorage.RAM)
```

Comment:
same comment as above.

Reply:
The answer is similar than above, with AutoComplete we are indexing and searching for both cases: 
- Dynamo execution
- Tests execution 
so for Node Autocomplete the writer needs to be disposed (doesn't matter if we are in test mode or not)